browser-interface/packages/entryPoints/index.ts

```diff
@@ -32,7 +33,14 @@ import { isWebGLCompatible } from './validations'
 declare const globalThis: { DecentralandKernel: IDecentralandKernel }
 globalThis.DecentralandKernel = {
   async initKernel(options: KernelOptions): Promise<KernelResult> {
-    SingleSignOn.init(SSO_URL)
+    try {
+      let sso = getSSOUrl()
+      if (sso) {
```

Comment:
should raise an error if the SSO is not initialized?

Reply:
No, should continue working in localStorage mode